src/components/StarRating/StarRating.scss

```diff
@@ -0,0 +1,31 @@
+$color-placeholder-star: #e1e1e1;
+$color-active-star: #d9af00;
+$color-number-reviews: gray;
+
+.CoveoStarRating {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+}
+
+.coveo-star-rating-star + .coveo-star-rating-star {
+    margin-left: 2px;
+}
+
+.coveo-star-rating-star-svg {
+    width: 15px;
+    height: 15px;
+    color: $color-placeholder-star;
+}
+
+.coveo-star-rating-star-active {
+    .coveo-star-rating-star-svg {
+        color: $color-active-star;
+    }
+}
+
+.coveo-star-rating-label {
+    margin-left: 5px;
+    font-size: 11px;
```

Comment:
Where thosemagic number comes from ?

Reply:
More pixels looks less good. less pixels looks less good. magic! :P
What kind of correlation are we usually looking for?

Reply:
why is the font-size 11px?
why is it 5px of margin-left?

You could use `em` instead perhaps?

Reply:
sure we can. It would make sense to size the stars according to em too then. 11px was chosen because it looked best with a fixed star size fo 16px.